utils/aoa-proxy/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2023 Jó Ágila Bitsch
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+ 
+PKG_NAME:=aoa-proxy
+PKG_VERSION:=0.2
+PKG_RELEASE:=1
+ 
+PKG_SOURCE:=aoa-proxy-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/jo-bitsch/$(PKG_NAME)/archive/refs/tags/v$(PKG_VERSION).tar.gz?
+PKG_HASH:=3a9dc94a3cd1b0cba1ae80396b21a6a63289f20dc426be7836834e5a3fc23a68
+
+PKG_MAINTAINER:=Jó Ágila Bitsch <jgilab@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+ 
+include $(INCLUDE_DIR)/package.mk
+
+TARGET_CFLAGS += -DNO_HID=1
+ 
+define Package/aoa-proxy
+  SECTION:=utils
+  CATEGORY:=Network
+  TITLE:=Interact with Android devices using the Android Open Accessory Protocol
+  URL:=https://github.com/jo-bitsch/aoa-proxy
+  DEPENDS:=+libusb-1.0
+  PKG_BUILD_DEPENDS:=+argp-standalone
```

Comment:
This does not belong here.

Reply:
Sorry, for the rookie mistake, moved it up. to the other section.